chromium_src/chrome/browser/unexpire_flags.cc

```diff
@@ -21,6 +21,7 @@ bool IsFlagExpired(const flags_ui::FlagsStorage* storage,
 #if BUILDFLAG(ENABLE_PLAYLIST)
   version_info::Channel channel = chrome::GetChannel();
```

Comment:
this is not used below.

Reply:
Oops, thanks. Changed define guard so that the variable isn't left unused.